.gitmodules

```diff
@@ -1,15 +1,16 @@
 [submodule "deps/jsstyle"]
-	path = deps/jsstyle
-	url = https://github.com/joyent/jsstyle.git
+    path = deps/jsstyle
+    url = https://github.com/joyent/jsstyle.git
 [submodule "deps/restdown"]
-	path = deps/restdown
-	url = https://github.com/trentm/restdown.git
+    path = deps/restdown
+    url = https://github.com/trentm/restdown.git
 [submodule "deps/javascriptlint"]
-	path = deps/javascriptlint
-	url = https://github.com/davepacheco/javascriptlint.git
+    path = deps/javascriptlint
+    url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/manta-scripts"]
-	path = deps/manta-scripts
-	url = https://github.com/joyent/manta-scripts.git
+    path = deps/manta-scripts
+    url = https://github.com/joyent/manta-scripts.git
+	branch = MANTA-1936
```

Comment:
This is only for testing and will be updated prior to merge after the corresponding `manta-scripts` PR is merged.